src/core/expr/fexpr_mean.cc

```diff
@@ -64,20 +66,23 @@ class FExpr_Mean : public FExpr_Func {
                             wf.get_frame_id(i),
                             wf.get_column_id(i)
                           );
-        Column coli = evaluate1(wf.retrieve_column(i), gby, is_grouped);        
+        Column coli = evaluate1(wf.retrieve_column(i), gby, has_by, is_grouped);        
```

Comment:
So why do we need to have `has_by`? The code was just fine before this commit, it was only the stype that was wrong in several tests.

Reply:
I guess I misunderstood then. So all void columns should return float64, whether or not a groupby is present?

Reply:
Yeah, there is no reason to distinguish.